packages/lodestar/src/util/httpClient.ts

```diff
@@ -0,0 +1,49 @@
+import Axios, {AxiosResponse, AxiosInstance} from "axios";
+
+export class HttpClient {
+  private _client: AxiosInstance;
+
+  public constructor(urlPrefix?: string) {
+    const _urlPrefix = urlPrefix? urlPrefix : "";
+    this._client = Axios.create({
+      baseURL: _urlPrefix
+    });
+  }
+
+  public get<T>(url: string): Promise<T> {
+    return new Promise((resolve, reject) => {
+      this._client.get<T>(url).then((result: AxiosResponse<T>) => {
```

Comment:
We are trying to avoid callbacks, we would prefer if you could use promisify library to avoid them